README.md

```diff
@@ -155,6 +155,48 @@ Which returns:
 <span title="my title">Hello, World!</span>
 ```
 
+##### Validation
+
+`ActionView::Component` extends with `ActionModel::Validations` by default. You can skip behaviour by configuring:
+
+`config/initializers/action_view_component.rb`:
+```ruby
+ActionView::Component.configure do |config|
+  config.extend_validations = false
+end
+```
+
+You are able to use any other validation libraries. To achieve that you can override `validate!` method:
+
+`app/contracts/profile_contract.rb`:
+```ruby
+class ProfileContract < Dry::Validation::Contract
+  params do
+    required(:email).filled(:string)
+  end
+end
+```
+
+`app/components/profile_component.rb`:
+```ruby
+class ProfileComponent < ActionView::Component::Base
+  def validate!
+    result = ProfileContract.new.call(email: email)
+
+    unless result.errors.empty?
+      raise RuntimeError.new(result.errors.to_h)
+    end
+  end
+
+  attr_reader :email
+  private :email
+
+  def initialize(email: nil)
+    @email = email
+  end
```

Comment:
its just documentation code but the constructor needs to be public.

Reply:
`initialize` is public. The only private method here is `def email`. Do you want to make it public?

Reply:
ok I misread the code